test/specs/step.js

```diff
@@ -0,0 +1,27 @@
+import { expect } from 'chai'
+import { clean, run, getResultFiles, getResultFileValue } from '../helper'
+
+describe('Step', () => {
+    beforeEach(clean)
+
+    it('should add step to the test', () => {
+        return run(['step']).then((results) => {
```

Comment:
Tests aren't checking that in the corresponding `test case` has created `step`

Reply:
added more tests